website/docs/audit.md

```diff
@@ -71,6 +71,10 @@ status:
     name: kube-system
 ```
 
+Limitations of getting violations from constraint status:
+
+- To reduce in-memory consumption of Gatekeeper audit pod and to avoid hitting [default etcd limit](https://etcd.io/docs/v3.5/dev-guide/limit/#request-size-limit) of 1.5MB per resource, gatekeeper recommends configuring a [limit up-to 500 violations](https://open-policy-agent.github.io/gatekeeper/website/docs/audit/#configuring-audit)(by default 20) on constraint template. Because of these limitations, users might not get all the violations from a Constraint resource.
```

Comment:
Why does this say “configuring a limit on constraint template”? The limit is a flag for the audit process that impacts number of violations that can be added to the status field of each constraint. Why is this on a constraint template?